### ISSUE TITLE
Panel icons do NOT appear on startup...

### ISSUE THREAD

**Issue by NIGHTWAV3** [home-sweet-gnome/dash-to-panel#1886]
**Things to do first**
- Confirm that the problem persists when Dash-to-Panel is the only enabled extension. To do so, disable every other extension, then restart gnome-shell by running the `r` command from the prompt that appears when pressing Alt+F2 on an X.org session, or by logging out/in on a Wayland session.
- Look for Dash-to-Panel errors in your log. To do so, run the `journalctl /usr/bin/gnome-shell -f -o cat &` command and reproduce the problem.
- Search existing opened and closed issues to see if the problem has already been reported.

**Describe the bug**
When starting up my machine, I can see that dash to panel is active, although my icons are all invisible. When I log out and log back in, I can see all of my icons again. I DO have custom icons (I changed their paths in the .desktop files in ``/usr/share/applications/`` directory, but that never was a problem on KDE, the icons always showed up. 

**Linux distribution and version**
Ubuntu 22.0.4

**GNOME Shell version**
Run `gnome-shell --version` from the command line to get this
![image](https://user-images.githubusercontent.com/131561567/234543908-9cca31c9-4b61-4b44-912c-c21979149554.png)

**Dash-to-Panel version**
This can be seen in the Dash-to-Panel Settings in the About tab.
![image](https://user-images.githubusercontent.com/131561567/234543961-a90244dd-a8ae-4694-b3d8-5b04c80218fd.png)

**Where was Dash-to-Panel installed from?**
The GNOME extensions website, GNOME Software store, your distribution's package manager, pre-installed with the distribution, directly from github, etc.
GNOME Extensions Browser panel

**Screenshots / Video captures**
If applicable, add screenshots or a link to a video capture to help explain your problem.
Basically self explanatory. My icons just don't show up when I start my machine, however, after relogging in, they appear. 

**Additional Info**
